Resources/Prototypes/Entities/Clothing/Head/helmets.yml

```diff
@@ -34,6 +34,18 @@
     sprite: Clothing/Head/Helmets/cult.rsi
   - type: IngestionBlocker
 
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHelmetEVA
+  name: EVA helmet
+  description: A basic helmet designed for extravehicular activies.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Helmets/eva.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Helmets/eva.rsi
+  - type: IngestionBlocker
```

Comment:
no pressure protection on the helmet?

Reply:
None of the helmets currently have pressure protection and it works fine without them. Look at the cosmonaut helmet.

Reply:
I added pressure to the helmet and now you can spacewalk with just the helmet and take no pressure damage.